src/sentinel.c

```diff
@@ -582,7 +582,7 @@ void sentinelEvent(int level, char *type, sentinelRedisInstance *ri,
     robj *channel, *payload;
 
     /* Handle %@ */
-    if (fmt[0] == '%' && fmt[1] == '@') {
+	if (fmt[0] == '%' && fmt[1] == '@' && ri != NULL) {
```

Comment:
This looks like an issue that's not specific to Windows. Could you elaborate on what the bug is? Would it make sense to submit this to antirez\redis?